test/unit/src/common/actions/t_auth-store.js

```diff
@@ -16,13 +16,16 @@ const getPackageUploadRequestMacaroon = proxyquire.noCallThru().load(
   '../../../../../src/common/actions/register-name',
   { 'localforage': localForageStub }
 ).getPackageUploadRequestMacaroon;
+
+const STORE_SERIES = '16';
```

Comment:
You could import this from `src/common/actions/register-name` instead by changing the code above this slightly:

    const registerNameModule = proxyquire.noCallThru().load(
      '../../../../../src/common/actions/register-name',
      { 'localforage': localForageStub }
    );
    const { getPackageUploadRequestMacaroon, STORE_SERIES } = registerNameModule;